migrations/sql/afterMigrate__1__seed_data.sql

```diff
@@ -933,7 +933,8 @@ UPDATE document_template SET form_spec_json = '[
       "label": "Bond Amount",
       "type": "FIELD",
       "placeholder": "Enter the bond amount",
-      "required": true
+      "required": true,
+      "context-props": ["currency"]
```

Comment:
adding another member seems a bit unnessecary, would it be easier to make the "type": "CURRENCY' and make a fieldtype of currency that extends the base field?

Reply:
I tried that at first, but if you look at how we get the currency functionality we don't pass props at that level, e.g.: 
```
            <Form.Item>
              <Field
                id="amount"
                name="amount"
                label="Bond Amount*"
                component={RenderField}
                {...currencyMask}
                validate={[required, number, currency]}
              />
            </Form.Item>
```

Reply:
It's extensible, but doesn't seem as maintainable given that the intent of what's being doing is not intuitive. I also don't like how it's blending concerns of very react-specific front-end component constructs verses the back-end defining the expectations of the form data. I agree with Jason and I would expect that whatever is processing the form definition acts on this type versus the back-end trying to tell the front-end how to work on this level.

Reply:
No problem. I have an alternate solution that I will implement then.